packages/extension-vscode/README.md

```diff
@@ -0,0 +1,13 @@
+# webhint extension for Visual Studio Code
+
+Generates diagnostics for workspace files based on webhint analysis.
+
+## Compile and Run
+
+* Run `npm install` in this folder.
+  This installs all necessary npm modules in the client and server folders.
+* Open VS Code on this folder.
+* Press Ctrl+Shift+B to compile the client and server.
```

Comment:
This needs to be updated for other OSes.

`ctrl` => `command` on macOS.

Reply:
Yep, it's also out-of-date since it references the client and server folders which have been merged. I'll fix it.